src/Adapter/HookManager.php

```diff
@@ -79,7 +80,11 @@ public function exec(
                 return Hook::exec($hook_name, $hook_args, $id_module, $array_return, $check_exceptions, $use_push, $id_shop);
             } catch (\Exception $e) {
                 $logger = ServiceLocator::get('\\PrestaShop\\PrestaShop\\Adapter\\LegacyLogger');
+                $environment = ServiceLocator::get('\\PrestaShop\\PrestaShop\\Adapter\\Environment');
```

Comment:
Oh right this is the old system of DI that was implemented in the early 1.7 versions ^^
We'll need to replace this with Symfony DI in the next major version, but it's not your job here 😉